src/Tools/ConfigurationSchemaGenerator/ConfigSchemaEmitter.cs

```diff
@@ -6,20 +6,24 @@
 using System.Text.Json;
 using System.Text.Json.Nodes;
 using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
 using System.Xml.Linq;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.DotnetRuntime.Extensions;
 using Microsoft.Extensions.Configuration.Binder.SourceGeneration;
 
 namespace ConfigurationSchemaGenerator;
 
-internal sealed class ConfigSchemaEmitter(SchemaGenerationSpec spec, Compilation compilation)
+internal sealed partial class ConfigSchemaEmitter(SchemaGenerationSpec spec, Compilation compilation)
 {
     private readonly TypeIndex _typeIndex = new TypeIndex(spec.AllTypes);
     private readonly Compilation _compilation = compilation;
     private readonly Stack<TypeSpec> _visitedTypes = new();
     private readonly string[] _exclusionPaths = CreateExclusionPaths(spec.ExclusionPaths);
 
+    [GeneratedRegex(@"( *)\r?\n( *)", RegexOptions.IgnoreCase)]
```

Comment:
What case is being ignored?

Reply:
Because I am dumb, copy pasted a sample and didn't pay attention.